PDOAssignment/UserPage.php

```diff
@@ -0,0 +1,28 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+	<title>UserHomePage</title>
+</head>
+<body>
+	<h1>Welcome : <?php include 'DatabaseConfig.php';  session_start(); echo "".$_SESSION['userName'];?></h1>
+	<hr>
+	<form align="right" method="POST" action="Logout.php"><input type="submit" name="logOut" value="Log Out!!!"></form>
+	
+	<h2>Your Books : <br>
+		<?php  
+			$selectStatement= "SELECT book_name FROM books WHERE user_email = '".$_SESSION['userName']."'"; 
+			$statement = $databaseConnection->prepare($selectStatement);
+			$statement->execute();
+			$result = $statement->fetchAll(PDO::FETCH_ASSOC);
+			foreach($result as $key=>$value)
```

Comment:
Always check if the data is not empty then only it should go in the loop. Check it everywhere in this PR